CHANGELOG.md

```diff
@@ -24,13 +28,28 @@ After this, `Decidim::Proposals::ProposalEndorsement` and the corresponding coun
 
 ### Changed
 
+- **decidim-comments**: Paginate comments [\#5553](https://github.com/decidim/decidim/pull/5553)
+
 ### Fixed
 
 - **decidim-proposals**: Fix relative path in mentioned proposal email [\#5852](https://github.com/decidim/decidim/pull/5852)
 - **decidim-proposals**: Use simple_format to add a wrapper to proposals body [\#5753](https://github.com/decidim/decidim/pull/5753)
 - **decidim-sortitions**: Fix incorrect proposals sortition. [\#5620](https://github.com/decidim/decidim/pull/5620)
 - **decidim-admin**: Fix: let components without step settings be added [\#5568](https://github.com/decidim/decidim/pull/5568)
 - **decidim-proposals**: Fix proposals that have their state not published [\#5832](https://github.com/decidim/decidim/pull/5832)
+- **decidim-comments**: Don't allow comments deeper than a certain depth, at the API level [\#5553](https://github.com/decidim/decidim/pull/5553)
+- **decidim-core**: Force SSL and HSTS [\#5553](https://github.com/decidim/decidim/pull/5553)
+- **decidim-api**: Make exposing GraphiQL optional[\#5553](https://github.com/decidim/decidim/pull/5553)
+- **decidim-api**: Do not expose GraphiQL in production [\#5553](https://github.com/decidim/decidim/pull/5553)
```

Comment:
As I see from the commits, this has been removed, so it shouldn't be here. Although I don't see the comment from @andreslucena saying that this should be removed, only that it should be disable by default? or did you have feedback from another channel?

Reply:
This was agreed with @decidim/product offline, sorry. You can check #5859 for an improved approach to the GraphiQL issue.